src/transformers/models/flava/modeling_flava.py

```diff
@@ -1949,15 +1949,14 @@ def forward(
 
                 if mim_labels is not None:
                     mim_labels = mim_labels[pos_mask]
+                    bool_masked_pos = bool_masked_pos[pos_mask]
```

Comment:
if pos mask is None this will unsqueese the bool masked pos, not expected I think. no?

Reply:
If I am not mistaken, it is not possible for pos_mask to be None at that point, because of line 1939

Reply:
Right sorry